### DISCUSSION TITLE
Cohesive Zone Analysis: FEM Solution is not a smooth curve

### DISCUSSION THREAD

**Discussion by AmanKumar0301** [idaholab/moose#28693] [Q&A Modules: Solid mechanics]
Hi all,
In cohesive zone method, I am trying to generate traction vs separation plot for mode-II loading. However the plot that I am getting is not a smooth curve. I am comparing it with the analytical plot of the traction separation law.
![Traction-Separation_Plot-1](https://github.com/user-attachments/assets/0c65cfa0-77f4-4e75-a7dd-cc779ba82900)

Can someone help what may be the reason. I have already played with the 'viscosity' parameter under CZM material.

Following is my input file:
![4](https://github.com/user-attachments/assets/3b4754f1-0a3f-46e1-aa6b-83e94e43198c)
![5](https://github.com/user-attachments/assets/19272a3c-d055-41cc-b07e-fc553298237d)
![6](https://github.com/user-attachments/assets/7ea14eb3-1018-4dc0-975f-e3fe31f04c2c)

@arovinelli @jiangwen84 @GiudGiud 

Is it something to do with the solver controls in executioner?

**Comment by GiudGiud** [comment]
Hello

This looks almost correct. Just a little bit of oscillations to remove. 
What does the convergence history look like? In the log of one of the noisy points.
Is there any nested solve in a material property that could be less converged?

**Reply by AmanKumar0301** [reply]
1. What does the convergence history look like? 
--> I am attaching the solution history somewhere between 1.6 seconds to 4 seconds when the curve has just started to drop in the first softening region.
![8](https://github.com/user-attachments/assets/5cdcabbd-4608-446c-8a48-e37f86f82494)
![9](https://github.com/user-attachments/assets/0ba2c008-1d39-4002-afa6-45cc31684fa3)

2. Just a little bit of oscillations to remove.
-->Yes I want to remove those oscillations. I want the FEM solution curve to be smooth. Because for other similar problems the oscillation are causing significant result difference. I am attaching a curve for one such comparative analysis. See the following:
![Traction-Separation_Plot-2](https://github.com/user-attachments/assets/d840e47b-57e5-4488-bc0b-74f08e9ddcbc)

3. Is there any nested solve in a material property that could be less converged?
I did not understand this part. However please see the script that I have attached.

Thanks for your interest.

**Reply by GiudGiud** [reply]
Oh I did not realize this was a single transient.

Then the oscillations could be coming from the time integration scheme. You are using the default 1st order implicit Euler. Let's try BDF2 (`scheme = bdf2` in the Executioner)

**Reply by GiudGiud** [reply]
except there is no time derivative term. So it's not that.

**Reply by AmanKumar0301** [reply]
![10](https://github.com/user-attachments/assets/0338adfb-0b7f-4c04-94ca-8ca32c0cf6bf)
I did try using scheme = bdf2 in the executioner.
But the results are exactly same. No change.
